src/App.vue

```diff
@@ -159,15 +170,15 @@ export default {
       AppStore.setSystemColorScheme(scheme.value);
     },
     attachStylesToRoot(CSSCustomProperties) {
-      const root = document.documentElement;
+      const root = document.body;
```

Comment:
I believe so, and I think this was one of the causes of the bugs with switching between modes. I agree that it would be more ideal to apply to the root instead of the body, but I think there were issues with implementing it that way in the original implementation to support dark mode, which is why it was done on the body. I think the main definitions happen in `src/styles/base/_colors.scss`.